src/pages/library/common.js

```diff
@@ -470,7 +470,7 @@ export const SearchAndFilterComponent = ({
             placeholder: 'Search Name or Description',
             itemToString: v => v[titleField],
             onChange: onSearchChange,
-            suggestionFilter: _.curry((needle, { lowerName, lowerDescription }) => _.includes(_.toLower(needle), `${lowerName} ${lowerDescription}`)),
+            suggestionFilter: _.curry((needle, listItem) => _.includes(_.toLower(needle), `${getLowerName(listItem)} ${getLowerDescription(listItem)}`)),
```

Comment:
One concern I have with these changes btw is that `normalizeDataset` would only be run once, right when the data is initially loaded. (Although I think we would have to check on this again, I don't quite remember the code.) This instance of `getLowerName` should be running every time the filter runs, right?

Long term, this could be a performance issue, i think?

Reply:
Yes, it definitely could be a performance issue. I think there's a bigger refactoring that can be done on this component to help address that while still maintaining use of our schema for the objects passed around, but I think that is out of scope for these changes. I _suspect_ that the performance issue that arrises first is going to be response time, which is probably going to lead us to service side filtering. But I am not 100% sure on that.

Reply:
I tested this out, and the performance breaks the text searching experience. Is there anything we can do to improve this?

Reply:
Actually, I take this back. After doing some profiling on localhost, it looks like the perf issue has something to do with the axe tools.